intake_thredds/source.py

```diff
@@ -69,6 +69,12 @@ def __init__(
         self.urlpath = url
         if 'simplecache::' in url:
             self.metadata.update({'fsspec_pre_url': 'simplecache::'})
+        if isinstance(path, str):
+            path = list(path)
+        if not isinstance(path, list):
+            raise ValueError(f'path must be list of str, found {type(path)}')
+        if not isinstance(path[0], str):
+            raise ValueError(f'path must be list of str, found {type(path)}')
```

Comment:
Do you mind adding one or two unit tests for this change?

Reply:
added

Reply:
Unfortunately, the unit tests appear not to cover these lines er [codecov](https://app.codecov.io/gh/NCAR/intake-thredds/blob/d1e4ca7906d9cbf75a4a7adfb018266f7e8c52d5/intake_thredds/source.py). I'm going to suggest changes shortly. Feel free to incorporate them

![Screen Shot 2022-01-11 at 11 18 29 AM](https://user-images.githubusercontent.com/13301940/148999084-4393fe3e-b38f-4d5c-9600-38b5e4a8c84a.png)